test/browsercontext-dsf.spec.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2018 Google Inc. All rights reserved.
+ * Modifications copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { it, expect, describe } from './fixtures';
+
+it('should fetch lodpi assets', async ({browser, server}) => {
+  const context = await browser.newContext({
```

Comment:
who cleans up this context? Do we track it in the browser fixture?

Reply:
also doesn't it need `...defaultContextOptions`?

Reply:
Ouch, thanks for catching it, I should not be using newContext at all, copied it from another test, did not copy footer.